image/pro_base/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM seafileltd/base:16.04
+
+# syslog-ng and syslog-forwarder would mess up the container stdout, not good
+# when debugging/upgrading.
+RUN apt update
```

Comment:
better run these statements in a single run command and cleanup at the end. We don't want to have an apt cache as part of the image or any layer of it